dango/warp/src/execute.rs

```diff
@@ -52,12 +61,26 @@ pub fn execute(ctx: MutableCtx, msg: ExecuteMsg) -> anyhow::Result<Response> {
     }
 }
 
+#[cfg_attr(not(feature = "library"), grug::export)]
+pub fn cron_execute(ctx: SudoCtx) -> anyhow::Result<Response> {
+    for (denom, rate_limit) in RATE_LIMIT
+        .range(ctx.storage, None, None, grug::Order::Ascending)
+        // Need to collect here because the iterator lock the ctx.storage.
+        .collect::<StdResult<Vec<_>>>()?
+    {
+        update_outbound_quota(ctx.storage, &ctx.querier, denom, rate_limit)?;
+    }
+
+    Ok(Response::new())
+}
+
 #[inline]
 fn set_route(
     ctx: MutableCtx,
     denom: Denom,
     destination_domain: Domain,
     route: Route,
+    rate_limit: Option<RateLimit>,
```

Comment:
Why is this necessary? The only case you invoke `set_route` is in `execute` where this is always `None`.